cli/recovery.c

```diff
@@ -977,8 +977,23 @@ static int dport_unlock(int argc, char **argv)
 		{NULL}
 	};
 
+	cfg.serial = 0;
+	cfg.unlock_version = 0xffff;
```

Comment:
It'd be much cleaner to initialize these in the structure variable definition. Like 

cfg = {
        .unlock_version = 0xffff;
};

In this way, no need to set serial to zero as it's done automatically.

Reply:
Updated. Please review.